app/forms/steps/capital/savings_form.rb

```diff
@@ -0,0 +1,56 @@
+module Steps
+  module Capital
+    class SavingsForm < Steps::BaseFormObject
+      delegate :saving_type, to: :record
+
+      attribute :provider_name, :string
+      attribute :account_balance, :pence
+      attribute :sort_code, :string
+      attribute :account_number, :string
+      attribute :is_overdrawn, :value_object, source: YesNoAnswer
+      attribute :are_wages_paid_into_account, :value_object, source: YesNoAnswer
+      attribute :account_holder, :value_object, source: OwnershipType
+
+      validates_presence_of(
+        :provider_name,
+        :sort_code,
```

Comment:
Ticket is asking for validation that SC is 6 digits and AN is 8 digits. Would need to strip '-' from SC, and squash any spaces if you do do that

Reply:
Oh I missed that on the ticket about validation. Let me check.

Reply:
Oh wait @timpeat it says 'There is _no_ validation' for SC and AN lengths - very confusing!

Reply:
I don't see how there can be validation as per current design. This form is used for all savings, not just UK bank accounts with a standard format.